src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -23,6 +23,7 @@ import dimen from '../../../utils/dimen';
 import {COLORS, FONTS, SIZES} from '../../../utils/theme';
 import {Context} from '../../../context/Store';
 import {RECENT_SEARCH_TERMS} from '../../../utils/cache/constant';
+import {colors} from '../../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `colors` has been added, but the old import statement for `COLORS` from `'../../../utils/theme'` is still present. If `COLORS` is no longer used in this file, it should be removed to avoid confusion and unnecessary imports.
```diff
- import {COLORS, FONTS, SIZES} from '../../../utils/theme';
+ import {FONTS, SIZES} from '../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->